src/openrct2/park/ParkFile.h

```diff
@@ -9,7 +9,7 @@ struct ObjectRepositoryItem;
 namespace OpenRCT2
 {
     // Current version that is saved.
-    constexpr uint32_t PARK_FILE_CURRENT_VERSION = 20;
+    constexpr uint32_t PARK_FILE_CURRENT_VERSION = 21;
 
     // The minimum version that is forwards compatible with the current version.
     constexpr uint32_t PARK_FILE_MIN_VERSION = 19;
```

Comment:
you'll need to bump the min version as the new save can't be read by version 20.